config/mkpj.sh

```diff
@@ -32,5 +32,5 @@ root="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
 config="${root}/config/prow/config.yaml"
 job_config_path="${root}/config/jobs"
 
-docker pull gcr.io/k8s-prow/mkpj || true
+dodocker pull gcr.io/k8s-prow/mkpj 1>&2 || true
```

Comment:
```suggestion
docker pull gcr.io/k8s-prow/mkpj 1>&2 || true
```

Reply:
just me fighting with me lagging laptop and writing a commit message in the editor, sigh. fixing